scripts/packaging/mac_distro.py

```diff
@@ -57,6 +57,11 @@ def check_rpath(home_path):
 
     success = True
 
+    darwinOpDylib = os.path.join(home_path,
+        'projects/robots/robotis/darwin-op/plugins/robot_windows/robotis-op2_window/librobotis-op2_window.dylib')
+    subprocess.run(["install_name_tool", "-rpath", "@loader_path/../../../../../../../",
+        "@loader_path/../../../../../../../../", darwinOpDylib])
```

Comment:
Also we generally use single quotes in Python:
```suggestion
    subprocess.run(['install_name_tool', '-rpath', '@loader_path/../../../../../../../',
        '@loader_path/../../../../../../../../', darwinOpDylib])
```